scavenger-collector/src/main/kotlin/com/navercorp/scavenger/config/JdbcConfig.kt

```diff
@@ -38,10 +38,9 @@ class JdbcConfig : AbstractJdbcConfiguration() {
         jdbcConverter: JdbcConverter
     ): SqlParameterSourceFactory {
         return EntityConvertibleSqlParameterSourceFactory(
-            ConvertibleParameterSourceFactory(DefaultJdbcParameterSourceConverter(), NoneFallbackParameterSource()),
+            ConvertibleParameterSourceFactory(DefaultJdbcParameterSourceConverter(Collections.emptyList()), NoneFallbackParameterSource()),
```

Comment:
how about using `emptyList()` of kotlin??

Reply:
fixed in 1b356ae56e11c7a8305fb34d98c0898c8ee9ae5f